README.md

```diff
@@ -66,7 +66,9 @@ targets to build Proton:
 `make install` - This will install Proton into your user's Steam directory.
 You may need to restart the Steam client to see it. The tool's name in the
 Steam client will be based on the currently checked out branch of Proton. You
-can override this name using the `build_name` variable.
+can override this name using the `build_name` make variable, which can be set 
+via an evironment variable or an argument to the make command 
```

Comment:
There's a small typo.
Change `eviroment` to `environment`.

Reply:
Thank you, I'll fix that.

Reply:
Cheers! 😄